docs/bank-accounts/Sandbox-Simulation.md

```diff
@@ -0,0 +1,15 @@
+---
+tags: [Bank Accounts]
+---
+
+# Sandbox simulation
+
+By default, when you create a bank account in Sandbox, it will move to the `Active` status almost immediately. You can change this behaviour to simulate other scenarios:
+
+## Suspended status
+
+If the bank account has the sort code `00-00-00` it will move to `Suspended`.
+
+## Pending status
+
+If the bank account has the sort code `00-00-01` it will stay in `Pending`.
```

Comment:
Not sure if its worth mentioning why it stays in pending / suspended? They will be able to see the amber / red bank checks in the dashboard